docs/release/release_0_4_19.md

```diff
@@ -0,0 +1,350 @@
+and generating black-to-color colormaps when adding layers to the viewer. 
+For example, one can now use: `viewer.add_image(data, colormap="crimson")` or `viewer.add_image(data, colormap="#FAFA00").
+To see what color names can be handled, please refer to the 
+[vispy` dictionary of colors](https://github.com/vispy/vispy/blob/269ed1ac4d8126421fd5a7eb06a2996d63f46b17/vispy/color/_color_dict.py#L181).
+Note: there is a namespace collision for the case of `orange` which is both a proper `vispy` colormap name and a color name. 
+Previously, `orange` was handled as the `vispy` colormap, which is from white to orange. 
+However, with 0.4.19, `orange` will be handled as a color and the generated colormap will be from black to orange. 
+
+The removal of public access to `viewer.window.qt_viewer` has been postponed to `v0.6.0` ([napari/napari/#6283](https://github.com/napari/napari/pull/6283)).
+If you're interested in some feature that's currently behind private or deprecated API,
+please make an issue on the napari repo requesting the feature to be exposed!
+
+napari is now pydantic 2 compatible. At this moment we are using `pydantic.v1` to achieve this. 
+Additionally, for this release, the napari conda bundle will be shipped with pydantic v1, but
+we expect that in the next release we will ship the bundle with pydantic v2.
+Please, if you are a plugin developer and your plugin uses pydantic, ensure
+that it is compatible with pydantic v2 ([napari/napari/#6358](https://github.com/napari/napari/pull/6358)).
+
+We have fixed problems with rendering some labels in incorrect color by sequence of Pull Requests:
+[napari/napari/#3308](https://github.com/napari/napari/pull/3308),
+[napari/napari/#6411](https://github.com/napari/napari/pull/6411),
+[napari/napari/#6467](https://github.com/napari/napari/pull/6467),
+[napari/napari/#6439](https://github.com/napari/napari/pull/6439).
+[napari/napari/#6607](https://github.com/napari/napari/pull/6607).
+This also improves performance of rendering labels.
+However, there are still some limitations.
+Best performance is for labels encoded as (u)int8.
+As GPU API accept 8 and 16 bits int textures and float textures,
+for labels stored as (u)int8 and (u)int16 we could transfer data to GPU without preprocessing.
+
+For bigger data types we need to convert data, so it leads to bigger memory usage.
+Also best performance for bigger data requires to install numba library.
+
+So if you wold like for best user performance, please try to use the minimal possible data type.
+Even if, in the future, we'll use more advanced techniques for rendering labels on GPU, that will allow
+transferring any data without preprocessing, the GPU often offers less VRAM that computer has RAM.
+Also we experiment a little with current options and increase the size of
+send data (uint8, uint16 and float32) decrease performance of rendering on GPU.
+The test was performed on NVIDIA GeForce RTX 3060 with 12GB VRAM.
```

Comment:
There's some info here that I don't think should be in the highlights. I would honestly just remove this whole chunk...